spec/fx/definition_spec.rb

```diff
@@ -29,13 +29,46 @@
         )
       end
 
+      it "performs ERB interpolation" do
+        sql_definition = <<-EOS
+          <% %w(test west).each do |function| %>
+            CREATE OR REPLACE FUNCTION <%= function %>()
+            RETURNS text AS $$
+            BEGIN
+              RETURN '<%= function %>';
+            END;
+            $$ LANGUAGE plpgsql;
+          <% end %>
+        EOS
+
+        erb_sql_definition = (<<-EOS).delete("\n").squeeze(' ').strip
+          CREATE OR REPLACE FUNCTION test()
+          RETURNS text AS $$
+          BEGIN
+            RETURN 'test';
+          END;
+          $$ LANGUAGE plpgsql;
+          CREATE OR REPLACE FUNCTION west()
+          RETURNS text AS $$
+          BEGIN
+            RETURN 'west';
+          END;
+          $$ LANGUAGE plpgsql;
+        EOS
+        allow(File).to receive(:read).and_return(sql_definition)
+
+        definition = Fx::Definition.new(name: "test", version: 1)
+
+        expect(definition.to_sql.delete("\n").squeeze(' ').strip).to eq erb_sql_definition
```

Comment:
This expectation is more than a mouthful to read. Not a blocker for now, but I'll take a stab at reformatting it to make it leaner and meaner